lib/Runtime/Types/JavascriptStaticEnumerator.cpp

```diff
@@ -49,6 +49,11 @@ namespace Js
         return this->prefixEnumerator == nullptr && this->arrayEnumerator == nullptr && this->propertyEnumerator.IsNullEnumerator();
     }
 
+    bool JavascriptStaticEnumerator::CanUseJITFastPath() const
+    {
+        return this->propertyEnumerator.CanUseJITFastPath() && this->currentEnumerator == nullptr;
```

Comment:
currentEnumerator is no less likely to be non-null, so I don't think it matters.